test

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -eu
+
+source ./build
+
+SRC="	src
+	src/config
+	src/exec
+	src/exec/stages
+	src/exec/stages/disks
+	src/exec/stages/files
+	src/exec/util
+	src/providers
+	src/providers/cmdline
+	src/providers/ec2
+	src/providers/file
+	src/providers/util
+	src/registry"
+
+echo "Running go fix..."
+fixRes=$(go tool fix -diff $SRC)
+
+echo "Checking gofmt..."
+fmtRes=$(gofmt -d -e -s $SRC)
+if [ -n "${fmtRes}" ]; then
+	echo -e "gofmt checking failed:\n${fmtRes}"
+	exit 255
```

Comment:
This script originally came from rkt and I've just been tweaking it since, that's where this came from.